src/response.spec.ts

```diff
@@ -113,7 +113,7 @@ describe('Response', () => {
 
   describe('#data', () => {
     it('returns the response data', () => {
-      expect(response.data<string>()).toEqual(responseData)
+      expect(response.data()).toEqual(responseData)
```

Comment:
What about a test that shows response typing or something under the typescript readme section? https://github.com/tulios/mappersmith#typescript

Reply:
Not sure, do you have a suggestion? I think most people would benefit for typing the client itself, this is just an improvement on that. E.g;
```
export interface MyClient {
  Healthcheck: {
    ping: () => Promise<Response<{ status: 'OK' | 'NOK' }>>
  }
}
const client: MyClient = {} as MyClient
// checkHealth: () => Promise<"OK" | "NOK">
const checkHealth = async () => {
  const x = await client.Healthcheck.ping()
  // can do x.status() or x.request() etc...
  return x.data().status
}
```